sql/src/main/java/io/crate/executor/transport/SnapshotRestoreDDLDispatcher.java

```diff
@@ -205,10 +205,11 @@ public void onFailure(Exception e) {
         return f;
     }
 
+    @VisibleForTesting
     static class ResolveIndicesAndTemplatesContext {
 
-        Collection<String> resolvedIndices = new HashSet<>();
-        Collection<String> resolvedTemplates = new HashSet<>();
+        final Collection<String> resolvedIndices = new HashSet<>();
```

Comment:
why aren't they private? can't we make them private? otherwise the add methods doesn't make so much sense or?